data/json/recipes/weapon/bashing.json

```diff
@@ -485,5 +485,22 @@
       { "proficiency": "prof_toolsmithing" }
     ],
     "components": [ [ [ "q_staff", 1 ] ] ]
+  },
+  {
+    "result": "knuckle_impact",
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_BASHING",
+    "skill_used": "fabrication",
+    "difficulty": 6,
+    "time": "8h",
+    "autolearn": true,
+    "using": [ [ "blacksmithing_standard", 20 ], [ "steel_standard", 8 ] ],
+    "proficiencies": [
+      { "proficiency": "prof_metalworking" },
+      { "proficiency": "prof_blacksmithing" },
```

Comment:
I would argue that "A pair of thick steel plates" doesn't require entire 8 hours of work for skilled armorsmith, nor joint articulation proficiency. It uses Outer layer.

Reply:
Joint articulation is used for the bracing to open and close onto your character's hand, and for the plates to stay dynamic as you move your fingers. I would argue most of the time spent is on the intricacies of making the bracing.